modules/system/applications.nix

```diff
@@ -37,11 +42,30 @@ in
         rm ~/Applications/'Nix Apps'
       fi
 
-      if [ ! -e '/Applications/Nix Apps' ] \
-         || ourLink '/Applications/Nix Apps'; then
-        ln -sfn ${cfg.build.applications}/Applications '/Applications/Nix Apps'
+      ourfolder () {
+        local marker
+        marker=$(xattr -p org.nix-darwin "$1" 2>/dev/null)
+        [ "$marker" = 'generated' ]
+      }
+
+      targetFolder='/Applications/Nix Apps'
```

Comment:
This is more of a future direction, but it would be nice if the `targetFolder` could be customized on a per-app basis. That would allow 1Password to work via nix-darwin. It currently expects to be in `/Applications` and doesn’t work anywhere else. (And if it could be used by Home Manager ….)

Reply:
Yeah, I want that too, as I have a fairly elaborate hierarchy in ~/Applications where I sort my apps by topic. Otherwise I wouldn't be able to find them again.

To keep this pull request focussed though, I would really like to push that out into future work.